### PR TITLE
doc: Update AUTHORS file

### PR DESCRIPTION
**Pull request by kaworu**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -226,6 +250,7 @@ Weilong Cui                             cuiwl@google.com
 Wenxian Li                              wofanli@gmail.com
 Will Deuschle                           wdeuschle@palantir.com
 Yongkun Gui                             ygui@google.com
+youssefazrak                            yazrak.tech@gmail.com
 Youssef Azrak                           yazrak.tech@gmail.com
```
Comment (kaworu) [inline AUTHORS]: duplicate, how this should be handled?

Comment (tklauser) [reply AUTHORS]: Probably best to add an entry to [.mailmap](../blob/master/.mailmap):

```
Youssef Azrak <yazrak.tech@gmail.com>
```

File: AUTHORS [hunk 2]
```diff
@@ -123,15 +135,18 @@ Koichiro Den                            den@klaipeden.com
 Kornilios Kourtis                       kornilios@isovalent.com
 Laurent Bernaille                       laurent.bernaille@datadoghq.com
 Lehner Florian                          dev@der-flo.net
+Lior Rozen                              liorrozen@users.noreply.github.com
```
Comment (tklauser) [inline AUTHORS]: Maybe add a `.mailmap` entry in order to get the real e-mail, see 14e611d72ead11ae268725e021f9766ce7f0fa88:

```
Lior Rozen <liorr@tailorbrands.com>
```

File: AUTHORS [hunk 3]
```diff
@@ -56,27 +60,34 @@ David Donchez                           donch@dailymotion.com
 Deepesh Pathak                          deepshpathak@gmail.com
 Devarshi Sathiya                        devarshisathiya5@gmail.com
 Dharma Bellamkonda                      dharma.bellamkonda@gmail.com
+Didier Durand                           durand.didier@gmail.com
 Diego Casati                            diego.casati@gmail.com
 Dmitry Kharitonov                       geakstr@me.com
 Dom Del Nano                            ddelnano@gmail.com
+elfadel                                 bonfohelfadel@gmail.com
```
Comment (tklauser) [inline AUTHORS]: Maybe add a `.mailmap` entry with the real name as per 82d03524a83262125177641f87f2d9bc24986aca's `Signed-off-by`:

```
El-Fadel Bonfoh <elfadel@accuknox.com>
```

File: AUTHORS [hunk 4]
```diff
@@ -123,15 +135,18 @@ Koichiro Den                            den@klaipeden.com
...
 Li Yi                                   denverdino@gmail.com
 m4rx0                                   m@footek.ch
+Maciej Fijalkowski                      maciej.fijalkowski@intel.com
 Maciej Kwiek                            maciej@isovalent.com
 Maciej Skrocki                          maciejskrocki@google.com
 Madhu Challa                            madhu@cilium.io
 MaiReo                                  sawako.saki@gmail.com
 Maksym Lushpenko                        iviakciivi@gmail.com
 Manali Bhutiyani                        manali@covalent.io
+mandarjog                               mandarjog@gmail.com
```
Comment (tklauser) [inline AUTHORS]: Might want to add a `.mailmap` entry with the name as well, see 53f35fbdd7dd730a496796665fd9c8730678efe2:

```
Mandar U Jog <mjog@google.com> <mandarjog@gmail.com>
```

File: AUTHORS [hunk 5]
```diff
@@ -169,18 +186,21 @@ Paweł Prażak                            pawelprazak@users.noreply.github.com
...
 Raghu Gyambavantha                      raghug@bld-ml-loan4.olympus.f5net.com
+Rahul Jadhav                            nyrahul@gmail.com
 Rajat Jindal                            rajatjindal83@gmail.com
 Ray Bejjani                             ray@isovalent.com
 Renat Tuktarov                          yandzeek@gmail.com
 Rene Zbinden                            rene.zbinden@postfinance.ch
 Robin Hahling                           robin.hahling@gw-computing.net
 Rodrigo Chacon                          rochacon@gmail.com
 Romain Lenglet                          rlenglet@google.com
+romanspb80                              romanspb@yahoo.com
```
Comment (tklauser) [inline AUTHORS]: Another possible entry for `.mailmap` (65dd482bac770a310f374c4b33fbee12fc4ef08a):

```
Roman Ptitcyn <romanspb@yahoo.com>
```

**Review by tklauser — CHANGES_REQUESTED** [review]
I  suggest a few more entries to `.mailmap` to get proper name/e-mail resolution for some contributors, see inline review comments.

**Comment by kaworu** [comment]
@tklauser thanks for the comment, I've fixed a couple of others on the way, please take another look!

File: AUTHORS (hunk 3 above)
Comment (tklauser) [reply AUTHORS]: Sorry, I think my suggested `.mailmap` entry was wrong. It should have said:

```
El-Fadel Bonfoh <elfadel@accuknox.com> <bonfohelfadel@gmail.com>
```

File: .mailmap [hunk 6]
```diff
@@ -18,7 +18,9 @@ Dan Wendlandt <dan@isovalent.com>
 Daniel Qian <qsj.daniel@gmail.com>
 Darshan Chaudhary <deathbullet@gmail.com>
 Devarshi Sathiya <devarshisathiya5@gmail.com>
+El-Fadel Bonfoh <elfadel@accuknox.com>
```
Comment (tklauser) [inline .mailmap]: I got this wrong in my suggested entry, it should instead be
```suggestion
El-Fadel Bonfoh <elfadel@accuknox.com> <bonfohelfadel@gmail.com>
```
as per my comment below.

**Review by tklauser — CHANGES_REQUESTED** [review]

**Review by tklauser — APPROVED** [review]

**Pull request merged**